sunpy/timeseries/timeseriesbase.py

```diff
@@ -121,6 +122,8 @@ def __init__(self, data, meta=None, units=None, **kwargs):
             # Should have a list of 3-tuples giving a complex metadata list.
             self.meta = meta
 
+        if isinstance(data.index[0], Time):
+            self._data.index = pd.to_datetime(self._data.index.map(lambda x: x.to_datetime()))
```

Comment:
elements of `self._data.index` is changed to `Pandas.Timestamp` because in

https://github.com/sunpy/sunpy/blob/782b242b06724ae315e9aad776c7673b895e3585/sunpy/timeseries/timeseriesbase.py#L249-L250

`Astropy.Time` does not support `round` function

Reply:
Why is the index an astropy time?